GetPosError

I'm trying to build pluto for the first time. What am I doing wrong?

```sh
~/clones/pluto% zig version
0.8.0-dev.1771+cc525bc00
~/clones/pluto% zig build
./build.zig:267:64: error: container 'std.fs.file.File' has no member called 'GetPosError'
    const Error = (error{EndOfStream} || File.ReadError || File.GetPosError || Allocator.Error || File.WriteError || File.OpenError);
                                                               ^
./build.zig:332:26: note: referenced here
    fn make(step: *Step) Error!void {
                         ^
./build.zig:355:88: note: referenced here
            .step = Step.init(.Custom, builder.fmt("Ramdisk", .{}), builder.allocator, make),
                                                                                       ^
~/clones/pluto%
```

I'm on Arch Linux.

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -8,7 +8,7 @@ on:
 jobs:
   test:
     name: Build mode ${{ matrix.build_mode }}
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
     strategy:
       matrix:
         build_mode: ["", -Drelease-fast, -Drelease-safe, -Drelease-small]

--- a/build.zig
+++ b/build.zig
@@ -264,7 +264,7 @@ const RamdiskStep = struct {
     out_file_path: []const u8,
 
     /// The possible errors for creating a ramdisk
-    const Error = (error{EndOfStream} || File.ReadError || File.GetPosError || Allocator.Error || File.WriteError || File.OpenError);
+    const Error = (error{EndOfStream} || File.ReadError || File.SeekError || Allocator.Error || File.WriteError || File.OpenError);
 
     ///
     /// Create and write the files to a raw ramdisk in the format:

